src/lib/web-component/filter-option/filter-option-element.js

```diff
@@ -42,10 +42,10 @@ export class FilterOption extends HTMLElement {
   addNewCheckbox(checkboxLabel) {
     const newCheckbox = this.checkboxTemplate.cloneNode(true);
     newCheckbox.querySelector('label span').textContent =
-      checkboxLabel || 'undefined';
+      checkboxLabel || 'other';
```

Comment:
Rather than duplicating the same string thrice, put it in a constant with a good name and use that instead.

Reply:
Added `otherLabel = `other'` at constructor and used this. :-)